src/@orb.yml

```diff
@@ -2,11 +2,13 @@ version: 2.1
 
 description: >
   Sample orb description
-  # What will your orb allow users to do?
-  # Descriptions should be short, simple, and clear.
+  # What will your orb allow users to accomplish?
+  # Descriptions should be short, simple, and imformative.
 
+# This information will be displayed in the orb registry and is not manditory.
```

Comment:
Should it be `mandatory`?

Reply:
@lokst If it was mandatory I would think that the orb would not validate. I would 100% make it mandatory for partners. For users with small personal projects, I don't know if we want to force this on them. I am open to either.

Reply:
I would think no unless not having it causes a validation error. For Partners it would be mandatory. For everyone it would be best practice.

The source URL I wouldn't mind saying is mandatory, maybe we could add that as a warning the future "strict" validation?

Reply:
@KyleTryon I agree with you, I actually meant to say that it looks like there is a mispelling of the word?